styles.css

```
@@ -14,18 +14,503 @@
 
 /* ------------------------------- 1. Global Styles -------------------------------- */
 
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+:root {
+    --purple: #5722CD;
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --black-main-text: #0C0C0C;
+    --light-purple: #F2F0FF;
+    --light-grey: #F3F6F9;
+    --grey: #7A7A7A;
+    --white: #FFF;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    color: var(--black-main-text);
+}
+
 /* ------------------------------- 2. Typography ----------------------------------- */
 
+h1, h3 {
+    font-family: 'Merriweather', sans-serif;
+}
+
+h2  {
+    font-size: 22px;
+}
+
 /* ------------------------------- 3. Layout --------------------------------------- */
 
 /* --------------- 3.1 Reusables ----------------------- */
 
+.outer-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+}
+
+.inner-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 700px;
```

Probeer `width` i.c.m. statische waardes te vermijden. Als je pagina nu heel smal maakt zie je het form voor een gedeelte niet meer. Wat je dan beter kan doen is een max-width (want dat is denk ik wat je bedoelt)
```suggestion
    max-width: 700px;
```